test/cql-pytest/suite.yaml

```diff
@@ -1,4 +1,4 @@
 type: Python
 pool_size: 4
 dirties_cluster:
-  - test_native_transport
+  - test_native_transport
```

Comment:
It looks like this change is unintentional

Reply:
dropped